views/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+!.gitignore
```

Comment:
I don't think you need to specify `.gitignore` because AFAIK `*` doesn't match files starting with `.`.

In any case, it seems more sensible IMHO to just put `views/` in the top-level `.gitignore` instead.

BTW, what is this folder for? Why it exists and why should it be ignored, some clarification in the commit message would be helpful!

Reply:
I need the .gitigonre because git ignores empty directories.
I cannot add views if it has no files.
And views need to exist for `-J` to work

Reply:
```make
__dummy_version := $(shell mkdir -p views && git describe --tags | cut -c2- | cut -f1-2 -d- > views/VERSION)
```

Reply:
I don't like this :-(

Reply:
Why?

Reply:
it looks hacky :)